### DISCUSSION TITLE
Help Us Sustain Open-vsx.org!

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1433] [General]
https://blogs.eclipse.org/post/john-kellerman/help-us-sustain-open-vsxorg

Damn!

## News from 23/04/11
Several organizations have expressed their interested to support OpenVSX. (Ericsson, Gitlab, Gitpod, Google, Huawei, Posit, SAP, STMicroelectronics)
So I think OpenVSX will be fine.

**Comment by daiyam** [comment]
[keep]

**Comment by xaberus** [comment]
@daiyam Just seen that... Are there any news about this? If I understand it correctly, there are a couple of other projects (with more funding than VScodium, i.e.,  coder/code-server, Gitpod, **Eclipse** Theia, **Eclipse** Che, **SAP Business Application Studio**) that depend on open-vsx.org?

EDIT: added the other forks...

**Reply by daiyam** [reply]
No, no news! But I've just emailed them to know more about.

**Reply by daiyam** [reply]
OK I've got an answer. I've updated my initial post.

**Comment by Mr-Spade** [comment]
What are the specific restrictions on the use of vscode's official marketplace in places other than Visual Studio and Visual Studio Code? If open-vsx.org cannot continue to be maintained, is it possible to maintain a list of extensions (like a json file) that are open source and can be well integrated with vscodium, and then download and install vsix from the official marketplace?

Of course, I still hope that open-vsx.org can continue to be maintained. Honestly, I'm not sure what "a new funding and governance model" is, it looks like they don't just want simple sponsorships or donations.

**Reply by daiyam** [reply]
It's due to the license of the Marketplace which limits its use to only MS product. (https://github.com/microsoft/vscode/issues/31168)

You can already do that with [VSIX Manager](https://open-vsx.org/extension/zokugun/vsix-manager).

https://github.com/coder/code-marketplace could become an alternative but I don't see any publisher management...
